modules/porous_flow/test/tests/fluidstate/tests

```diff
@@ -4,69 +4,102 @@
     input = 'waterncg.i'
     exodiff = 'waterncg_liquid.e'
     threading = '!pthreads'
+    requirement = 'PorousFlowWaterNCG shall correctly calculate the properties of the liquid phase'
```

Comment:
You should not refer to class names in `requirements`. Requirements are first, then design where class names appear. That's what you have correctly in `design`.  You want to say "Porous flow module shall calculate...". 

Also, I would not use the word "correctly" in a requirement (think about this statement "I want to compute an incorrect value of quantity XYZ" - see?).